language/stdlib/modules/0L/LibraAccount.move

```diff
@@ -942,20 +940,17 @@ module LibraAccount {
         payer_sig: &signer,
         payee: address
     ) acquires LibraAccount, Balance, AccountOperationsCapability {
-        let bootstrap_amount = 1000000; // 1 gascoin, without scaling representation 
         let payer_addr = Signer::address_of(payer_sig);
         let account_balance = borrow_global_mut<Balance<Token>>(payer_addr);
         let balance_coin = &mut account_balance.coin;
-
-        assert(Libra::value(balance_coin) > bootstrap_amount * 2, Errors::limit_exceeded(EINSUFFICIENT_BALANCE));
+        assert(Libra::value(balance_coin) > BOOTSTRAP_COIN_VALUE, Errors::limit_exceeded(EINSUFFICIENT_BALANCE));
         // Should abort if the 
```

Comment:
Can we assert that payee is a validator here?

Reply:
Also, the balance_coin should not become 0 after they transfer the value. Can we ensure it is 2x?

Reply:
> Can we assert that payee is a validator here?
Yes probably, good idea.

> Also, the balance_coin should not become 0 after they transfer the value. Can we ensure it is 2x?
Yes I think there are multiple checks in there for that.